cmd/kubeadm/app/util/runtime/runtime_windows.go

```diff
@@ -0,0 +1,38 @@
+// +build windows
+
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	winio "github.com/Microsoft/go-winio"
+)
+
+const (
+	dockerSocket     = "//./pipe/docker_engine"         // The Docker socket is not CRI compatible
+	containerdSocket = "//./pipe/containerd-containerd" // Proposed containerd named pipe for Windows
```

Comment:
@PatrickLang  is the ContainerD pipe correct?

Reply:
According to [this](https://github.com/containerd/containerd/blob/2088fc999c20cb9759b0b537276e608f918e7c16/defaults/defaults_windows.go#L37) - yes.
However, what is more concerning to me are the unix path separators being used here vs windows ones. Is this correct?

Reply:
It worked fine with the forward slashes but I changed them to backslashes

Reply:
Reverted back to forward slashes per discussion below